ambuda/admin.py

```diff
@@ -8,11 +8,17 @@
 import ambuda.database as db
 import ambuda.queries as q
 
+from ambuda.logger import Logger
+
+LOGGER = Logger()
+
 
 class AmbudaIndexView(AdminIndexView):
     @expose("/")
     def index(self):
+        LOGGER.debug("Checking if current user is admin...")
         if not current_user.is_admin:
+            LOGGER.error("User is not admin")
```

Comment:
Since the logs are only on our local machine (not returned), it should be safe to have them (under the assumption that no one has access to the local logs). If this is still unsafe / not preferred, we can remove these logs. I mainly used them as a quick way to test logging.

Reply:
I don't think it's unsafe, but I think these logs are extraneous / not very useful and it's best to go ahead and remove them?